addon/components/frost-select-dropdown.js

```diff
@@ -196,6 +215,25 @@ export default Component.extend({
       return {
         className: classNames.join(' '),
         label: get(item, 'label'),
+        secondaryLabels: get(item, 'secondaryLabels'),
+
+        @readOnly
+        @computed('secondaryLabels')
+        hasSecondaryLabels (secondaryLabels) {
```

Comment:
Is there is specific reason to use a cp here instead of using simply…
`hasSecondayLabels: isArray(secondaryLabels)`